docs/logging-monitoring/logging-architecture.rst

```diff
@@ -0,0 +1,41 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Logging and Monitoring architecture
+===================================
+
+Airflow supports a variety of logging and monitoring mechanisms as shown below.
+
+.. image:: ../img/arch-diag-logging.png
+
+By default, Airflow supports logging into the local file system. These include logs from the Web server, the Scheduler, and the Workers running tasks. This is suitable for development environments and for quick debugging.
+
+For cloud deployments, Airflow also has hooks contributed by the Community for logging to cloud storage such as AWS, GCP, and Azure.
```

Comment:
GCP => Google Cloud
See:  https://developers.google.com/style/word-list#gcp

Reply:
Done, good catch

Reply:
Can you create a new PR?

Reply:
done